HTML/Default.sublime-keymap

```diff
@@ -0,0 +1,10 @@
+[
+	{ "keys": [">"], "command": "move", "args": {"by": "characters", "forward": true}, "context":
```

Comment:
Should this binding respect `auto_match_enabled` like the bindings for `)'"]}`?

Reply:
Well, what prompted this was the attribute completions that add `”>`. Those aren’t triggered by auto_match_enabled, although I do realize the overwriting of “ is controlled by that setting.

Reply:
I wonder if we'd need a separate setting (e.g.: `smart_type_enabled`) to enable such kind of features, which are related with `auto_match_enabled` only. Some of the new bindings in CSS which auto-add spaces to maintain code style would be a use case, too.

Reply:
I'm going to kick the can down the road for now. We can spend some time later thinking about potential settings that affect overwriting, but aren't strictly related to auto-matching.